client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/JmxUtil2.java

```diff
@@ -0,0 +1,62 @@
+package org.infinispan.client.hotrod.impl;
+
+
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+
+import org.infinispan.commons.logging.Log;
+import org.infinispan.commons.logging.LogFactory;
+
+/**
+ * Internal class containing JMX related utility methods. Strictly for internal use. This class has just become unstable
+ * and might evaporate spontaneously.
+ *
+ * @author Galder Zamarreño
+ * @since 5.0
+ * @deprecated Since 10.0. DO NOT USE! Will be moved to an internal package soon.
+ */
+@Deprecated
+public final class JmxUtil2 {
```

Comment:
I guess it could be renamed to `JmxUtil` and undeprecated

Reply:
It was just an experiment to see what CI thinks about it, I just removed it entirely. The code directly calls MBeanServer.register/unregisterMBean. But now I'm not so sure